doc/source/deployment/setup-deployer.rst

```diff
@@ -176,7 +155,3 @@ You can improve SSH connections by enabling pipelining:
    [ssh_connection]
    pipelining = True
    EOF
-
-
-With this done, continue your deployment with the
-:ref:`configuredeployment` page.
```

Comment:
is this part supposed to be removed?

Reply:
this is the only page with such an instruction. The deployment guide has SES Integration as the following step. And the Configure Cloud instructions indicate that the output from SES Integration must be present in order to Configure for SES Integration.